Formula/git-gui.rb

```diff
@@ -17,13 +17,6 @@ class GitGui < Formula
 
   depends_on "tcl-tk"
 
-  # Patch to fix Homebrew/homebrew-core#68798.
-  # Remove at version bump
-  patch do
-    url "https://github.com/git/git/commit/1db62e44b7ec93b6654271ef34065b31496cd02e.patch?full_index=1"
-    sha256 "0c7816ee9c8ddd7aa38aa29541c9138997650713bce67bdef501b1de0b50f539"
-  end
```

Comment:
The patch hasn't made it to the latest release, so it looks like it's still needed: https://github.com/git/git/blob/773e25afc41b1b6533fa9ae2cd825d0b4a697fad/gitk-git/gitk#L12661

Reply:
Can you add a link to the associated PR in a comment? That way we have a better idea of when it can be removed.

https://github.com/git/git/pull/944